src/test/kotlin/org/jmailen/gradle/kotlinter/functional/CustomTaskTest.kt

```diff
@@ -52,6 +52,7 @@ class CustomTaskTest : WithGradleTest.Kotlin() {
                         println("go")
                     }
                 }
+
```

Comment:
A new (I think? haven't looked into it) rule failed with `no newline at end of file`.  Perhaps that was a valid test and the rule is disabled somewhere? Will take a look